operator/config/deploy/karmada-operator-deployment.yaml

```diff
@@ -21,13 +21,11 @@ spec:
         imagePullPolicy: IfNotPresent
         command:
         - /bin/karmada-operator
-        - --kubeconfig=/etc/config
+        - --leader-elect-resource-namespace=karmada-system
         - --v=4
-        volumeMounts:
-        - name: kubeconfig
-          mountPath: /etc/config
-          subPath: config
-      volumes:
-      - name: kubeconfig
-        secret:
-          secretName: my-kubeconfig
+        ports:
+        - containerPort: 8080
+          name: metrics
+          protocol: TCP
+      serviceAccount: karmada-operator
+      serviceAccountName: karmada-operator
```

Comment:
ServiceAccount is a depreciated alias for ServiceAccountName. Can it be removed?

Reply:
Agree, removed.